### ISSUE TITLE
Securely sharing social media credentials between members

### ISSUE THREAD

**Issue by duksis** [elixir-berlin/elixir-berlin.github.io#1]
Would be great to increase the meetups activity on Twitter, Facebook, G+ to make elixir in Berlin more viral.
For this would be good if more members would have access to posting on those channels.

Some ways we can share the credentials

1. emailing or instant messaging them around (not very secure, and loads of hassle keeping them updated)
2. Shared keepass file on google drive or dropbox

Do you guys have any other suggestions for this? @slashmili @mindreframer @iurifq

**Comment by mindreframer** [comment]
This can be arbitrarily complex…

A solution:- shared keepass file on a private bitbucket repository.

BTW: a team on bitbucket with some private repos does not hurt

Other plentiful alternatives:-
https://spin.atomicobject.com/2014/06/25/diy-password-manager-gpg-git/-
https://news.ycombinator.com/item?id=8264496-
https://github.com/lanmaster53/sharepass- https://github.com/ahoward/sekrets-
https://github.com/StackExchange/blackbox
But… Well. We dont need enterprise-grade security, I guess.
Cheers!
 





On Mon, Dec 5, 2016 12:52 PM, Hugo Duksis notifications@github.com
wrote:
Would be great to increase the meetups activity on Twitter, Facebook, G+ to make
elixir in Berlin more viral.
For this would be good if more members would have access to posting on those
channels.

Some ways we can share the credentials

 1. emailing or instant messaging them around (not very secure, and loads of
    hassle keeping them updated)
 2. Shared keepass file on google drive or dropbox

Do you guys have any other suggestions for this? @slashmili  @mindreframer  
@iurifq

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.



------------------------------  Roman Heinrich
Emailroman.heinrich@gmail.com  Githubgithub.com/mindreframer
Twitter@mindreframer
Blogdevopsbox.es  Xingxing.to/mindreframer  Linkedin
de.linkedin.com/in/mindreframer  Resumedevopsbox.es/resume/resume.html 
Delicious delicious.com/mindreframer  Skypemindreframer

**Comment by slashmili** [comment]
I'm for `shared keepass file on a private bitbucket repository.`

**Comment by iurifq** [comment]
I agree with @mindreframer. Top security is not really the point here. A bitbucket repo seems like a good option to start it off.

**Comment by mindreframer** [comment]
I have created a team on Bitbucket:
https://bitbucket.org/account/user/elixirberlin
And you are all admins
Cheers!
 





On Mon, Dec 5, 2016 5:18 PM, Iuri Fernandes notifications@github.com
wrote:
I agree with @mindreframer. Top security is not really the point here. A
bitbucket repo seems like a good option to start it off.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.



------------------------------  Roman Heinrich
Emailroman.heinrich@gmail.com  Githubgithub.com/mindreframer
Twitter@mindreframer
Blogdevopsbox.es  Xingxing.to/mindreframer  Linkedin
de.linkedin.com/in/mindreframer  Resumedevopsbox.es/resume/resume.html 
Delicious delicious.com/mindreframer  Skypemindreframer

**Comment by duksis** [comment]
Here is the repo https://bitbucket.org/elixirberlin/credentials with twitter credentials 
I don't really know how facebook and G+ work, but if there are any shared access credentals @slashmili can you add them plase?

master key in slack

**Comment by slashmili** [comment]
For FB and G+ I only need to add you as admin and that should be enough.

**Comment by duksis** [comment]
oh then twitter is the only one for now - please add me

**Comment by slashmili** [comment]
Just join 
G+: https://plus.google.com/u/2/communities/107831462311035535095
FB: https://www.facebook.com/groups/237714459982331/

And I'll grant you access
